### PR TITLE
feat(#5): wraps telepresence in ike develop cmd

### PR DESCRIPTION
**Pull request by bartoszmajsak**
```
Usage:
  ike develop [flags]

Flags:
  -d, --deployment string   name of the deployment or deployment config
  -h, --help                help for develop
  -m, --method string       telepresence proxying mode - see https://www.telepresence.io/reference/methods (default "inject-tcp")
  -p, --port int            port to be exposed (default 8000)
  -r, --run string          command to run your application
```

### CODE REVIEW

File: cmd/ike/main.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ func main() {
 	// implementing the logr.Logger interface. This logger will
 	// be propagated through the whole operator, generating
 	// uniform and structured logs.
-	logf.SetLogger(logf.ZapLogger(false))
+	logf.SetLogger(logf.ZapLogger(true))
```
Comment (aslakknutsen) [inline cmd/ike/main.go]: Need a better way to handle this as we want the Service to deploy using 'false'

File: cmd/ike/cmd/develop.go [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+const telepresenceBin = "telepresence"
+
+var developCmd = &cobra.Command{
+	Use:   "develop",
+	Short: "starts the development flow",
+	Long:  `TODO`,
+	Run: func(cmd *cobra.Command, args []string) {
+
+		checkIfTelepresenceExists()
```
Comment (aslakknutsen) [inline cmd/ike/cmd/develop.go]: Shouldn't this 'stop' the further execution?

File: cmd/ike/cmd/develop.go [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+		log.V(9).Info("Failed to copy either of stdout or stderr")
+	}
+
+	wg.Wait()
+}
+
+func parseArguments() []string {
+	runArgs := strings.Split(runnable, " ")
+	return append([]string{
+		"--swap-deployment", deploymentName,
```
Comment (aslakknutsen) [inline cmd/ike/cmd/develop.go]: Should we write down an issue on not using swap from the client, but having the service prepare the 'correct' image?

File: cmd/ike/main.go (hunk 1 above)
Comment (bartoszmajsak) [reply cmd/ike/main.go]: You want that to be fixed as part of this PR? Maybe we need two separated things - CLI and that thing which runs on the server? That's probably a decision which needs to be discussed in a bit more detail. For now I'm fine with `json` output - can revert it to the previous state and make an issue for the potential split. WDYT?

File: cmd/ike/cmd/develop.go (hunk 2 above)
Comment (bartoszmajsak) [reply cmd/ike/cmd/develop.go]: Right, with `logrus` it did without explicit `exit` call. Added in 9f08970

File: cmd/ike/cmd/develop.go (hunk 3 above)
Comment (bartoszmajsak) [reply cmd/ike/cmd/develop.go]: I can take care of it today.

File: cmd/ike/main.go (hunk 1 above)
Comment (aslakknutsen) [reply cmd/ike/main.go]: sounds good.

File: cmd/ike/cmd/develop.go [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+		"--method", method,
+		"--run"}, runArgs...)
+}
+
+func checkIfTelepresenceExists() {
+	path, err := exec.LookPath(telepresenceBin)
+	if err != nil {
+		log.Error(err, fmt.Sprintf("Couldn't find '%s' installed in your system.\n"+
+			"Head over to https://www.telepresence.io/reference/install for installation instructions.\n", telepresenceBin))
+		os.Exit(1)
```
Comment (aslakknutsen) [inline cmd/ike/cmd/develop.go]: ... technically I would return an error and handle the exit in the caller.. or rename the function to explicitly 'warn' about the exit happening.

Comment (bartoszmajsak) [reply cmd/ike/cmd/develop.go]: How about meeting mid-way - ded1887?

File: cmd/ike/cmd/develop.go (hunk 3 above)
Comment (bartoszmajsak) [reply cmd/ike/cmd/develop.go]: Created epic #10

**Pull request merged**